Gruntfile.js

```diff
@@ -205,22 +205,12 @@ module.exports = function (grunt) {
                     'src/js/study-browser/eva-study-browser-grid.js',
                     'src/js/views/eva-study-view.js',
                     'src/js/views/eva-variant-view.js',
-                    'src/js/views/eva-gene-view.js',
```

Comment:
I had forgotten about this view. In order to compensate its disappearance, we could add some links from the annotation tab:

`Ensembl Gene ID` pointing to http://www.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000139618
`Ensembl Gene Symbol` pointing to http://www.ensembl.org/Homo_sapiens/Gene/Summary?g=BRCA2 (automatic redirection to the equivalent gene ID page)
`Ensembl Transcript ID` pointing to http://www.ensembl.org/Homo_sapiens/Transcript/Summary?t=ENST00000530893